.github/ISSUE_TEMPLATE/theme-submission.md

```diff
@@ -6,7 +6,9 @@ about: Theme submission
 
 # Theme submission
 
-**Please make sure that you have read the [theme submission guidelines](https://github.com/gohugoio/hugoThemes/blob/master/README.md#adding-a-theme-to-the-list) before submitting a theme**. The guidelines provide all relevant information and requirements that **have to be fulfilled** before the submission. Please tick the relevant boxes for your theme in the checklist below:
+**Please make sure that you have read the [theme submission guidelines](https://github.com/gohugoio/hugoThemes/blob/master/README.md#adding-a-theme-to-the-list) before submitting a theme**. The guidelines provide all relevant information and requirements that **have to be fulfilled** before the submission. We strongly suggest that you test your theme with the [Hugo Themes Build Script](https://github.com/gohugoio/hugoThemes/blob/master/README.md#testing-a-theme-with-the-hugo-themes-website-build-script) before submitting the theme for review. If a submission does not meet the criteria mentioned in the README it will be closed. You may re-submit once you fix problems with your submission. However, please note that we are not a free help center for general web development and for Hugo support questions there is a dedicated [support forum](https://discourse.gohugo.io/) that you should use.
```

Comment:
> However, please note that we are not a free help center for general web development and for Hugo support questions there is a dedicated [support forum](https://discourse.gohugo.io/) that you should use.

I suggest to rephrase this section slightly as follows:

However, please note that we have limited ressources and can not provide help for general web development issues. For support questions please refer to Hugo's dedicated [support forum](https://discourse.gohugo.io/) instead.